### DISCUSSION TITLE
Simple Post request without render or redirect

### DISCUSSION THREAD

**Discussion by Muetze42** [inertiajs/inertia#1155] [Help (Vue)] [answered]
I just want a simple post request with Inertia. 
The user must also be authenticated in the controller.
But I want this request without callback, render or forwarding. 

Is this possible somehow with inertia?

**Comment by t1m0t** [comment, marked as answer]
@Muetze42 you can do simple `fetch()` I believe. You would need to include the proper headers for authenticated requests. For that, you can inspect the request from the browser and adapt accordingly. For incoming request to you server, use the logic you need as it is agnostic as regards to inertia.

Inertia provides a way to properly render your app with either vue/swelte/react and adds some helpers to ease communication between client and server but you can build your own way if it doesn't fit your use cases.

Could you also provide an example as there are other possible answers?

**Comment by Muetze42** [comment]
OK. Currently I just created an auth with token and use Axios....

**Comment by usmanibrahim74** [comment]
@Muetze42 is there any way to do it with inertia? I just want to do a silent call that should not trigger anything. I can use fetch or axios, but preferring not to do
